test/logflare/source/billing_writer_test.exs

```diff
@@ -13,14 +15,14 @@ defmodule Logflare.Source.BillingWriterTest do
     user = user |> Logflare.Repo.preload(:billing_account)
     plan = insert(:plan, type: "metered")
 
-    pid =
-      start_supervised!(
+    {:ok, pid} =
+      start_supervised(
         {BillingWriter,
          %RecentLogsServer{source_id: source.token, source: source, user: user, plan: plan}}
       )
 
     # increase log count
-    start_supervised!(Logflare.Sources.Counters)
+    start_supervised(Logflare.Sources.Counters)
```

Comment:
this would silently fail

Reply:
It's ok to leave it as is. This was failing with `:already_started` errors, but if it fails to startup, the test will catch it.

Other than this approach, we could check if the process exists and only start at that point but that felt bloated for a test.